app/controllers/mainCtrl.js

```diff
@@ -1,36 +1,52 @@
 (function ()
 {
     'use strict';
-    angular.module('cinkciarz')
-            .controller('MainController', MainController);
-    function MainController(CurrencyService)
+
+    function MainController(CurrencyService, $localStorage)
     {
         var ctrl = this;
 
-        ctrl.message = "Trener Cienkciarza";
+        ctrl.message = 'Trener Cinkciarza';
 
         ctrl.data = {
-            model: 0,
-            availableOptions: [
-                {id: '1', name: 'USD'},
-                {id: '2', name: 'EUR'},
-                {id: '3', name: 'CHF'},
-                {id: '4', name: 'GBP'}
+            model: null, availableOptions: [{name: 'USD'}, {name: 'EUR'}, {name: 'CHF'}, {name: 'GBP'}
 
             ]
         };
 
+
+        ctrl.getCurrency = function (selectedValue)
+        {
+            CurrencyService.getCurrency(selectedValue)
+                    .then(function (result)
+                    {
+                        ctrl.money = result;
+                        ctrl.moneyInSell = result;
+                    });
+        };
+
+        ctrl.wallet = $localStorage.$default([{pln: ctrl.amount}, {eur: 0}, {dol: 0}, {chf: 0}, {gbp: 0}]);
+
         ctrl.amount = 0;
 
-         CurrencyService.getCurrency('usd')
-                .then(function (result)
-                {
-                    console.log(result);
-                    ctrl.dolar = result;
-                });
+        ctrl.allCurrencies = function (value)
```

Comment:
The name of this function (`allCurrencies`) is misleading. Inside you `get` only one currency not all.